### PR TITLE
Completed javascript-arrays

### PR DESCRIPTION
**Pull request by jooliecodes**
Screenshot 2023-08-28 at 8 49 00 PM

### CODE REVIEW

File: javascript-arrays/javascript-arrays-quiz-notes.md [hunk 1]
```diff
@@ -0,0 +1,45 @@
...
+
+- Describe array literal notation.
+
+- How are arrays different from "plain" objects?
+
+- What number represents the first index of an array?
+
+- What is the `length` property of an array?
+
+- How do you calculate the last index of an array?
```
Comment (shawnkost) [inline javascript-arrays/javascript-arrays-quiz-notes.md]: Don't forget to answer the quiz questions!

**Review by shawnkost — CHANGES_REQUESTED** [review]
Need to answer the quiz questions!

File: javascript-arrays/javascript-arrays-quiz-notes.md (hunk 1 above)
Comment (jooliecodes) [reply javascript-arrays/javascript-arrays-quiz-notes.md]: sorry, forgot to push them!

**Review by shawnkost — APPROVED** [review]
👍🏻

**Pull request merged**